README.adoc

```diff
@@ -316,13 +329,16 @@ Results :
 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
 ----
 
-To see whether the tests detect a failure, change the base URL in the configuration file so that when the `inventory` service tries to access the invalid URL, an UnknownUrlException is thrown.
+To see whether the tests detect a failure, change the base URL in the configuration file so that when the `inventory` service tries to access the invalid URL, an UnknownUriException is thrown.
 Rerun the Maven build. You see a test failure occur.
 
 == Great work! You're done!
 
 You just invoked a remote service by using a template interface with MicroProfile Rest Client in Open Liberty.
 
+Although not covered in this guide, MicroProfile Rest Client provides a uniform way to configure SSL for the client.
+You can learn more in the https://openliberty.io/blog/2019/06/21/microprofile-rest-client-19006.html#ssl[Hostname verification with SSL on Open Liberty and MicroProfile Rest Client^] blog and the https://github.com/eclipse/microprofile-rest-client/releases[MicroProfile Rest Client specification^].
```

Comment:
```suggestion
You can learn more in the https://openliberty.io/blog/2019/06/21/microprofile-rest-client-19006.html#ssl[Hostname verification with SSL on Open Liberty and MicroProfile Rest Client^] blog and the https://github.com/eclipse/microprofile-rest-client/releases/tag/1.3.4[MicroProfile Rest Client 1.3 specification^].
```

I'd suggest using the more specific 1.3.4 release page since it includes the details on SSL config.

Reply:
Actually for guides, we generally don't use specific versions. We point to the `release` link for the spec, user can refer to the latest spec available at the time.

Reply:
Sounds good. I'll mark this as resolved.